src/server/v2.0/handler/auditlog.go

```diff
@@ -44,7 +43,7 @@ func (a *auditlogAPI) ListAuditLogs(ctx context.Context, params auditlog.ListAud
 		return a.SendError(ctx, err)
 	}
 
-	if !secCtx.IsSysAdmin() {
+	if err := a.RequireSysAdminAccess(ctx, rbac.ActionList, rbac.ResourceAuditLog); err != nil {
```

Comment:
Should only accept ForbiddenError, other errors need to return.

Reply:
no logic change here, and this API requires authentication. 

Discussed offline, not a problem.